packages/dara-components/js/common/select/select.tsx

```diff
@@ -38,9 +38,6 @@ export function getMultiselectItems(values: Array<any>, items: Array<Item>): Arr
     }, []);
 }
 
-const toItem = (value: any): Item | undefined | null =>
-    typeof value === 'string' || typeof value === 'number' ? { label: String(value), value } : value;
-
```

Comment:
I think it was intended that this did not look up existing items here. The use case was for persisting selected value when the item was no longer (perhaps temporarily) in the list. I'd speak to @Roman-Kornev about it as I think he was implementing it for our internal product here https://github.com/causalens/dara/pull/123, let's make sure it still serves that purpose

Reply:
Oh I see, will update so that if the value is not present it can still be shown

Reply:
Yes, that's correct. It was for preserving values if they are not present in the list